### ISSUE TITLE
BUG: Found Error-code Pop up after Attempting to Submit 2 Reservations with the same venue, date, and time range. 1st res submit successful, 2nd res submit- error code pops up. (Suggestion, add Warning Prompts - This is regarding to avoid schedule conflicts of two reservations)

### ISSUE THREAD

**Issue by ralch34** [philrhoy/IMCCVenueReservationSystem2023#33]
NEW ISSUE... IF I ATTEMPT to submit a 2nd reservation with details that have the same venue, date, and time range from the 1st reservation I submitted earlier... Naay error code mugawas on the 2nd res submission.

![Screenshot_27 WARNING WE NEED TO FIX THIS](https://github.com/philrhoy/IMCCVenueReservationSystem2023/assets/90671718/0c30ddbe-6bef-4349-b2d1-08d7db862883)

Suggestion:
If user attempts to submit a reservation, unknowingly naa nay approve reservation nakauna sa Venue, Date n Time,
and after clicking "Submit"  - a prompt box appears.

"Warning: This reservation has conflicted with an Approved Reservation RES03123, conflict detected!"
(System Displays RES03123 Venue and Sched Details)     
"You cannot proceed unless choose another venue, date, or time range".
"back to edit" nga button dayon.

"Warning: This reservation has conflicted with a Pending Reservation RES09822, conflict detected!"
(System Displays RES09822 Venue and Sched Details)
"You cannot proceed unless choose another venue, date, or time range".
"back to edit" nga button dayon.

**Comment by ralch34** [comment]
Tested... It works.... issue Address.. Thanks dear developers *heart *heart XD
![Screenshot_15](https://github.com/philrhoy/IMCCVenueReservationSystem2023/assets/90671718/93504b97-1041-48a2-bcfa-457d311c8c2a)
![Screenshot_17](https://github.com/philrhoy/IMCCVenueReservationSystem2023/assets/90671718/f45d4604-625e-47b6-b459-ce554db40b2e)
